totalRP3_Extended/locale.lua

```diff
@@ -1647,14 +1647,18 @@ You can also use Total RP 3: Extended variables (|cffcccccc${my_var_name}|r).
 	--- THEN MOVE IT UP ONCE IMPORTED
 	------------------------------------------------------------------------------------------------
 
-	OP_OP_DATE_DAY = "Date: Day";
-	OP_OP_DATE_DAY_TT = "The current day, local time.";
-	OP_OP_DATE_MONTH = "Date: Month";
-	OP_OP_DATE_MONTH_TT = "The current month, local time.";
-	OP_OP_DATE_YEAR = "Date: Year";
-	OP_OP_DATE_YEAR_TT = "The current year, local time.";
-	OP_OP_DATE_DAY_OF_WEEK = "Date: Day of the week";
-	OP_OP_DATE_DAY_OF_WEEK_TT = "The current day of the week, local time. From 1 (Sunday) to 7 (Saturday).";
+	EFFECT_SOUND_ID_FADEOUT = "Fadeout duration (optional)",
+	EFFECT_SOUND_ID_FADEOUT_TT = "The duration (in seconds) over which the sound will be stopped.\n\nLeave empty to stop immediately.",
+	EFFECT_SOUND_ID_STOP_FADEOUT_PREVIEW = "Stops sound ID %s in channel %s over %s seconds.",
+	EFFECT_SOUND_ID_STOP_FADEOUT_ALL_PREVIEW = "Stops |cff00ff00all|cffffff00 sounds in channel %s over %s seconds.",
```

Comment:
Shouldn't this be `|cff00ff00all|r` instead of starting a new colour sequence?

Reply:
I copied the previous effect which was written before the color unpiling, but good point